TFG-Labs/custom-shipping-step-by-items
# Feature / TLVA-7

### What problem is this solving?

Functionality developed to satisfy customer requirements for "television" products requiring their license.

#### How it works

If there is a product from the "television" category in the shopping cart, the customer will be shown a form that must be filled in.

The data from this form will be stored with the address in Masterdata.

#### How to test it?

[Workspace](https://tlva7b--thefoschini.myvtex.com/)

You must be logged in with a customer already registered on the platform.

In the checkout add the product with SKU **1**, which has category "television".

When you reach the Shipping step and you have entered the shipping address, the form will appear as you can see in the following video

### Screenshots/Video example usage

[Video](https://www.loom.com/share/fb47a26e599643a09c7a41a76b25147a)

#### How does this PR make you feel? 🔗

![](https://media.giphy.com/media/VbAFrrDVGAvZu/giphy-downsized-large.gif)
